plugins/README.md

```diff
@@ -36,6 +37,22 @@ Tips:
 -   To see a trace of logs and wasm ABI calls, add `--loglevel=TRACE`.
 -   Optionally specify plugin config data using the `--config=<path>` flag.
 
+# Building Plugins
+To build your plugin using bazel, run:
+
+```
+bazelisk build <BUILD Target>
+```
+For example, to build the sample add_header plugin in C++, from the `plugins/` subdirectory of this repository, run:
```

Comment:
optional: shorten to:

"For example, to build the sample plugin in the `plugins/add_header` subdirectory, run:"